### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by aliesbelik** [aliesbelik/jmeter-amqp-plugin#52] [Announcements]
Minor release with some new features, bug fixes, UI improvements, and dependency upgrades.

### Added

* `App_id` message property for AMQP Publisher (see jlavallee#37) by @aliesbelik in #29.
* `Timestamp` message property for AMQP Publisher (see jlavallee#37) by @aliesbelik in #50.
* Help link on the title panel in AMQP Publisher and Consumer samplers.

### Fixed

* Fix error on JMeter startup caused by manifest absence by @AbstractBike in #48.
* setResponseData removed from AMQP Publisher.
* setSamplerData removed from AMQP Consumer.

### Infra

* Release action workflow added.
* Java 21 added to CI workflow.

### Dependency Updates

* Up **jmeter-core** version to 5.6.2.
* Up **amqp-client** version to 5.18.0.

## New Contributors

* @AbstractBike made their first contribution in #48

**Full Changelog**: https://github.com/aliesbelik/jmeter-amqp-plugin/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/aliesbelik/jmeter-amqp-plugin/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by ganeles** [comment]
Hello.
Firstly, thank you for your plugin, it's really perfect!

But I want to ask you about consumer sampler:
1) Why it clears the queue if I turn on "AutoACK" checkbox?
It looks as this checkbox enable "Purge queue" too

2) I thought, prefetch count means number of messages that should be got from the queue, but it doesn't. How does it works?

3) what does mean "Use transaction" 

And - thank you again for the nice plugin that helps me a lot!
